src/Person.jsx

```diff
@@ -0,0 +1,36 @@
+import './App.scss';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried,
+    partnerName = 'I am not married' } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        My name is
+        {' '}
+        {name}
+      </h2>
+      <p className="Person__age">
+        {age && (
```

Comment:
Don't render this element if there is no age 
```suggestion
      {age && (
        <p className="Person__age">
```